playground/backend/pkg/api/pathchecker/pathchecker.go

```diff
@@ -0,0 +1,49 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Could you please clarify why the pathchecker is located in the API folder? if it is not a mistake - I'm afraid I have to disagree with that, better to store it in the executor or create a new package like "checkers."

Reply:
moved to fs_tool package